omniNotes/src/androidTest/java/it/feio/android/omninotes/tests/AppointmentDetailsTest.java

```diff
@@ -0,0 +1,100 @@
+
+        // Step: Click "Add" button
+        Home.clickAddButton();
+
+        // Verify: Test "Checklist" is between 2 other texts
+        assertTrue("CheckList is not above TextNote", Home.isCheckListAboveTextNote());
+        assertTrue("CheckList is not Below Photo", Home.isCheckListBelowPhoto());
+
+        // Verify: The suggestive picture of "Photo" text is above "Checklist" suggestive picture
+        assertTrue("PicturePhoto is not above PictureCheckList", Home.isPicturePhotoAbovePictureCheckList());
+
+        // Step: Click "Text note" button
+        Home.clickTextNoteButton();
+
+        // Verify: "Content" and "Add reminder" are descendant of a LinearLayout and they are siblings
+        assertTrue("Content is not descendant of Linear Layout", AddTextNote.isContentBoxDescendantOfLinearLayout());
+        assertTrue("Add Reminder is not descendant of Linear Layout", AddTextNote.isAddReminderDescendantOfLinearLayout());
+        assertTrue("Content and Add reminder are not siblings", AddTextNote.isContentParentSiblingWithAddReminderParent());
+
+        // Verify: "Content Container text has a font size of 16sp;
+        assertEquals("Content text doesn't have a 16sp font size", 16, AddTextNote.getExpectedTextSizeOfContentContainer());
+        //Same result, different approach
+        assertTrue("Content text doesn't have a 16sp font size", AddTextNote.isFontSizeOfContentContainerExpectedSize());
+
+        // Step: Click "Title" container
+        AddTextNote.clickTitleContainer();
+
+        // Step: Type "Doctor appointment"
+        AddTextNote.typeDoctorAppointment();
+
+        // Step:Click "Content" container
+        AddTextNote.clickContentContainer();
+
+        // Step: Type "Tuesday, 17 Jul 2018, 18:30." 3 times
+        AddTextNote.typeDateAndTime();
+
+        //Verify: Content container has 2 lines of text
+        assertEquals("Content container doesn't have 2 lines of text", 2, AddTextNote.getExpectedNoOfLinesOfContentContainer());
+        //Same result, different approach
+        assertTrue("Content container doesn't have 2 lines of text", AddTextNote.isContentContainerTextOnExpectedNoOfLines());
+
+        // Step: Click on "Add reminder"
+        AddTextNote.clickAddReminder();
+
+        // Step: Add a reminder for the date from "Content" container
+        AddTextNote.clickOnADay();
+
+        // Step: Click "Done" button
+        AddTextNote.clickDone();
+
+        // Verify: The "Time picker" is displayed
+        assertTrue("Time picker is not displayed", AddTextNote.isTimePickerDisplayed());
+
+        // Step: Click "Done" button
+        AddTextNote.clickDoneButton();
+
+        // Step: Swipe the "repeat weekly" button
+        AddTextNote.enableRepeatWeekly();
+
+        // Verify: The number of weeks is between 2 texts
+        assertTrue("The number of weeks is not between 2 texts", AddTextNote.isWeekNoRightOfEveryText());
```

Comment:
You could say "The number of weeks is not on the right of Every text"

Reply:
Done